en/SetupView/Joystick.md

```diff
@@ -43,6 +43,30 @@ The following joysticks/controllers have been shown to work with relatively rece
 
 These are both highly recommended. They work well "out of the box" and have many buttons that you can map to flight modes. 
 
+#### Sony PS4 - DualShock 4 Controller V2
+
+This type allows both USB and wireless connection (Bluetooth). Wired USB connection usually works out of the box. The wireless connection needs some setup. 
+
+##### Linux Ubuntu setup
```

Comment:
Thanks @kaklik !

@DonLakeFlyer Can we automate this bit of setup as part of the installation? I'd rather not include developer instructions in the user guide.

Reply:
I don't know anything about DualShock wireless setup. I'd assume this refers to setup with respect to the OS which QGC would have nothing to do with.